code/game/objects/items/storage/firstaid.dm

```diff
@@ -114,7 +114,7 @@
 	generate_items_inside(items_inside,src)
 
 /obj/item/storage/firstaid/ancient
-	icon_state = "firstaid"
+	icon_state = "oldifrstaid" // yes this is the icon state its typo'd i know stop asking

```

Comment:
Just fix it?

Reply:
i wanted to avoid conflicts but sure

Reply:
See what i mean rohesie?

Reply:
Merge master into your branch first. After you've done that, rename it.
And done, won't be conflicts.

Reply:
I can't follow basic instructions and everything is fucked now